comptests/ha/test_restart_pod.py

```diff
@@ -240,6 +337,8 @@ def test_serverpod_status_online_after_delete_deployment(self):
         Publish the pod online event to Hare -
         server pod comes online after server pod restart using deployment
         """
+
+        # This Tc will fail for now - need fix CORTX-29300
```

Comment:
do u want to add a skip marker instead with the defect ID wherever applicable?

Reply:
Done